diff --git a/mono/mini/aot-runtime-wasm.c b/mono/mini/aot-runtime-wasm.c
index ccc5a26510c..8f26a725abd 100644
--- a/mono/mini/aot-runtime-wasm.c
+++ b/mono/mini/aot-runtime-wasm.c
@@ -63,13 +63,7 @@ handle_enum:
 	}
 }
 
-#if TARGET_SIZEOF_VOID_P == 4
-#define FIDX(x) ((x) * 2)
-#else
 #define FIDX(x) (x)
-#endif
-
-
 
 typedef union {
 	gint64 l;
diff --git a/mono/mini/interp/interp.c b/mono/mini/interp/interp.c
index 46b1664f078..b19b88d29c9 100644
--- a/mono/mini/interp/interp.c
+++ b/mono/mini/interp/interp.c
@@ -1384,11 +1384,7 @@ static InterpMethodArguments* build_args_from_sig (MonoMethodSignature *sig, Int
 #if DEBUG_INTERP
 			g_print ("build_args_from_sig: margs->fargs [%d]: %p (%f) (frame @ %d)\n", int_f, margs->fargs [int_f], margs->fargs [int_f], i);
 #endif
-#if SIZEOF_VOID_P == 4
-			int_f += 2;
-#else
-			int_f++;
-#endif
+			int_f ++;
 			break;
 		default:
 			g_error ("build_args_from_sig: not implemented yet (2): 0x%x\n", ptype);
